Engine/src/simulation/properties/action/impl/proximity/ProximityAction.java

```diff
@@ -24,13 +24,13 @@ public Object getValue() {
         return null;
     }
 
-    public void Invoke(EntityInstance firstEntityInstance, EntityInstance secondEntityInstance, EntityInstance[][] grid, int lastChangeTickCount) {
+    public void invoke(EntityInstance firstEntityInstance, EntityInstance secondEntityInstance, EntityInstance[][] grid, int lastChangeTickCount) {
```

Comment:
If we implement a 'Grid' object we need to send this object to proximity's invoke, and not the two dimensional array.

Reply:
And done!